tests/saver/test_save_prediction.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import pandas as pd
+
+from scipy.stats import norm
+
+from cascade.saver.save_prediction import uncertainty_from_prediction_draws, _predicted_to_uploadable_format
+
+
+def test_uncertainty_from_prediction_draws():
+    rng = np.random.RandomState(4398221)
+
+    num_draws = 100
+    locations = [1, 2]
+    age_lower, age_upper = zip(*[(0, 0.5), (50, 60)])
+    sexes = [-0.5]
+    time_lower, time_upper = zip(*[(y, y) for y in [2000, 2020]])
+    integrands = ["prevalence", "Sincidence"]
+
+    df = pd.DataFrame(
+        index=pd.MultiIndex.from_product(
+            [locations, age_lower, age_upper, time_lower, time_upper, integrands, sexes],
+            names=["location", "age_lower", "age_upper", "time_lower", "time_upper", "integrand", "s_sex"],
+        )
+    ).reset_index()
+
+    true_mean = np.linspace(0.0001, 2.0, len(df))
+    rng.shuffle(true_mean)
+    true_std = np.linspace(0.00001, 0.001, len(df))
+    rng.shuffle(true_std)
```

Comment:
You're using rng!

Reply:
I want a local seed! Also this is the pattern rust encourages so I've gotten used to it.

Reply:
I'm happy about the rng use. That's what I meant to say. And seeding is the right thing.